tools/Ecoregionalization_workflow/Nb_cluster.xml

```diff
@@ -0,0 +1,103 @@
+<tool id="Cluster_Estimate" name="ClusterEstimate" version="0.1.0+galaxy0" profile="22.05">
+    <description>Find an optimal number of cluster with SIH index </description>
+    <requirements>
+       <requirement type="package" version="4.2.3">r-base</requirement>
+       <requirement type="package" version="2.1.4">r-cluster</requirement>
+       <requirement type="package" version="1.1.1">r-dplyr</requirement>
+       <requirement type="package" version="2.0.0">r-tidyverse</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+         '$__tool_directory__/nb_clust_G.R'
+         '$input1'
+         '$input2'
+         '$input3'
+         '$max_k'
+         '$metric'
+         '$sample'
+         '$output1'
+         '$output2'
+         '$output3'
+    ]]>
+    </command>
+    <inputs>
+      <param name="input1" type="data" format="txt,csv,tabular" label="Environment file"/>
+      <param name="input2" type="data" format="txt" label="Taxa selected file (List of taxa from previous step)"/>
```

Comment:
```suggestion
      <param name="input2" type="data" format="txt" label="Taxa selected file" help="List of taxa from previous step"/>
```

What is the previous tool name, that would be more meaningful.

Reply:
Is such a taxa file not a standard? Or can you not use a standard?

Reply:
not a standard

Reply:
done